app/src/organisms/GripperWizardFlows/index.tsx

```diff
@@ -66,6 +66,10 @@ export function GripperWizardFlows(
     },
   })
   const [isExiting, setIsExiting] = React.useState<boolean>(false)
+  const [errorMessage, setShowErrorMessage] = React.useState<string | null>(
```

Comment:
If there is no specific reason to use `setShowErrorMessage`
```suggestion
  const [errorMessage, setErrorMessage] = React.useState<string | null>(
```